### PR TITLE
Feat/cc 145 advantages landing page

### PR DESCRIPTION
**Pull request by PengchengYuGithub**

### CODE REVIEW

File: components/Advantages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Image from "next/image";
+import { useInView } from "react-intersection-observer";
+const Advantages = () => {
+  const { ref, inView } = useInView();
+  const advantages = ["Online tools", "Customise", "High quality", "Years guarantee"];
+
+  return (
+    <section
+      className="max-w-7xl mx-auto pt-10 pb-20 px-3 md:px-6 lg:px-12 g-4 md:g-8 bg-purpleBlue text-white font-light overflow-hidden"
```
Comment (tatoMa) [inline components/Advantages/index.tsx]: What is g-4 and md:g-8?

File: components/Advantages/index.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      </h2>
+      <p className={`text-p opacity-85 tracking-[.01em] ${inView && "animate-advantageFadeIn85"}`}>
+        We provide high quality product for everything you need
+      </p>
+      <div className="grid grid-cols-2 md:flex gap-2.5 mt-4 md:mt-8 tracking-[.01em]">
+        {advantages.map((item, index) => {
+          return (
+            <div
+              key={index}
+              className={`md:w-1/4 h-44 px-4 py-10 gap-2 rounded-lg bg-backgroundColor flex flex-col items-center ${
```
Comment (tatoMa) [inline components/Advantages/index.tsx]: Try w-full instead of md:w-1/4.

File: components/Advantages/index.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                inView && (index < 2 ? "animate-advantageToRight" : "animate-advantageToLeft")
+              }`}
+            >
+              <Image src={`/Advantage${index + 1}.svg`} width={64} height={64} alt="advantage" />
```
Comment (tatoMa) [inline components/Advantages/index.tsx]: The alt can be an item string instead of a firm one "advantage".

File: tailwind.config.js [hunk 4]
```diff
@@ -67,6 +68,24 @@ module.exports = {
...
           "100%": { transform: "translateX(0)" },
         },
+        fadeIn: {
+          "0%": { opacity: "0", transform: "translateY(-1px)" },
+          "95%": { opacity: "1", transform: "translateY(-1px)" },
+          "100%": { opacity: "1", transform: "translateY(0px)" },
+        },
+        fadeIn85: {
+          "0%": { opacity: "0", transform: "translateY(-1px)" },
+          "95%": { opacity: "0.85", transform: "translateY(-1px)" },
```
Comment (tatoMa) [inline tailwind.config.js]: I dont get it. Why need to translate the Y point with -1 pixel?

Comment (PengchengYuGithub) [reply tailwind.config.js]: Maybe I understand it incorrectly: when I play the "Flow 2" in Figma, there seems a slight drop for the position of the title after fade-in. If it is not the designated purpose I will just remove this Y point translate.

**Review by tatoMa — APPROVED** [review]
LGTM

**Pull request merged**